css/style.css

```diff
@@ -0,0 +1,1140 @@
+  font-size: 12px;
+  line-height: 16px;
+  color: #36b37e;
+  margin: 10px 5px;
+  text-align: center;
+}
+
+.detail-tags li {
+  background: #ebf0ee;
+  border-radius: 4px;
+}
+
+.menu-body ul li {
+  width: 351px;
+  height: 56px;
+  border-radius: 8px;
+  margin-left: 28px;
+  margin-top: 40px;
+  font-size: 24px;
+  font-weight: 600;
+  cursor: pointer;
+}
+
+.menu-body a {
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #3a4a42;
+}
+
+.headline {
+  width: 375px;
+  height: 726px;
+  background: #fff url(../images/header-illsutration-mobile.png) no-repeat 100%;
+  background-size: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 64px 24px;
+  gap: 58px;
+  position: relative;
+  left: 0;
+  top: 92px;
+  overflow: hidden;
+}
+
+.header-title {
+  height: 192px;
+  display: flex;
+  flex-direction: column;
+  padding: 0 24px 0 0;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 35px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.h-span {
+  text-align: center;
+  color: #237855;
+}
+
+.heder-info {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+  width: 327px;
+  height: 164px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.social-link {
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin: 0 auto;
+}
+
+.social-link a {
+  width: 375px;
+  height: 189px;
+  margin-top: 19px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  color: #42526e;
+  text-align: center;
+}
+
+.medias {
+  display: flex;
+  list-style: none;
+  justify-content: center;
+  margin: 14px;
+  max-width: fit-content;
+}
+
+.medias i:hover {
+  transform: scale(2);
+  transition: all 0.5s ease-out;
+}
+
+.head-btn button {
+  width: 40px;
+  height: 40px;
+  padding: 8px;
+  left: 50%;
+  bottom: 100px;
+  position: absolute;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 32px;
+}
+
+.head-btn button:hover {
+  cursor: pointer;
+  background-color: black;
+  opacity: 0.7;
+  color: #fff;
+}
+
+.img-btn {
+  max-width: 100%;
+}
+
+.work-card {
+  width: 375px;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.myrecentwork {
+  width: 327px;
+  height: 97px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  position: relative;
+  text-align: center;
+  color: #172b4d;
+}
+
+.ind {
+  width: 48px;
+  height: 4px;
+  background: #36b37e;
+  border-radius: 24px;
+  margin: 24px 160px;
+}
+
+.work-container-card {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] The project guidelines require you to use `display: grid` to align your work-cards. Kindly change that.